libs/components/src/accordion/vl-accordion.component.ts

```diff
@@ -34,7 +35,7 @@ export class VlAccordionComponent extends BaseElementOfType(HTMLElement) {
     }
 
     static get _observedClassAttributes() {
-        return ['bold', 'disabled'];
+        return ['bold', 'disabled', 'is-standaard-open'];
```

Comment:
Door dit attribuut toe te voegen aan de observedClassAttributes gaat deze toegevoegd worden aan het class-attribuut van het component, dit is niet de bedoeling.

Reply:
just!